test/integration/use_int_test.go

```diff
@@ -160,6 +160,53 @@ func (suite *UseIntegrationTestSuite) TestReset() {
 	}
 }
 
+func (suite *UseIntegrationTestSuite) TestShow() {
+	suite.OnlyRunForTags(tagsuite.Use)
+
+	ts := e2e.New(suite.T(), false)
+	defer ts.Close()
+
+	cp := ts.SpawnWithOpts(e2e.WithArgs("use", "show"))
+	cp.Expect("No default project is set")
+	cp.ExpectExitCode(1)
+
+	cp = ts.SpawnWithOpts(
+		e2e.WithArgs("checkout", "ActiveState-CLI/Python3"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.Expect("Checked out Python3")
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(
+		e2e.WithArgs("use", "ActiveState-CLI/Python3"),
+		e2e.AppendEnv("ACTIVESTATE_CLI_DISABLE_RUNTIME=false"),
+	)
+	cp.Expect("Switched to Python3")
+	cp.ExpectExitCode(0)
+
+	cp = ts.SpawnWithOpts(e2e.WithArgs("use", "show"))
+	cp.ExpectLongString("The default project to use is ActiveState-CLI/Python3, located at")
+	if runtime.GOOS != "windows" {
+		projectDir := filepath.Join(ts.Dirs.Work, "Python3")
+		cp.ExpectLongString(projectDir)
+	} else {
+		// Windows sometimes uses shortened paths, sometimes not.
+		// Just verify the last two components of the path are there and assume all is well,
+		// especially if the Linux and macOS tests are passing.
+		projectDir := filepath.Join(filepath.Base(ts.Dirs.Work), "Python3")
```

Comment:
nit: We have `fileutils.GetLongPath` and `fileutils.GetShortPath`, one of them might help with this.

Reply:
That's good to know, but there's still the matter of knowing *which* one Windows is going to use for a given path.

Reply:
We could just verify that either longpath or shortpath matches. That's a safer assertion than just using the base dir.